src/niri.rs

```diff
@@ -2077,6 +2106,50 @@ impl Niri {
             .cloned()
     }
 
+    pub fn output_by_name(&self, name: Option<&str>) -> Option<Output> {
+        let Some(name) = name else {
+            return None;
+        };
+        self.global_space
+            .outputs()
+            .find(|output| &output.name() == name)
```

Comment:
I recently changed this to be case-insensitive in the rest of the code; this should be updated accordingly I guess.